### DISCUSSION TITLE
UICatalog+ xshell or putty terminal + ssh to debian console

### DISCUSSION THREAD

**Discussion by Winsor** [tui-cs/Terminal.Gui#2322] [General]
Hello All!
There is such a problem:
Build and Publish UICatalog demo as Release|net 6.0|linux-x64 configuration from Windows directly to linux-box. Main app window in the terminal does not work correctly with any of the color schemes in xshell or putty default configuration. 
![My console...](https://user-images.githubusercontent.com/3974713/217013708-e1757126-343b-4028-9456-51344ee6cfb8.png)
Demo correctly work only with -usc arg. But also with some problems, example of a cursor in the menu.
![Work...](https://user-images.githubusercontent.com/3974713/217014848-9e500c91-b1a5-4c44-95c2-69af08fa600d.png)

Can Is it possible to make CursesDriver work correctly? and what exactly is the problem (putty settings, publish settings)? themselves ncurses-demo work correctly in the same terminal.
Thanks for any information.

**Comment by BDisp** [comment]
See #1455. Maybe you have to set `xterm-256color` on the `Putty`.

**Reply by Winsor** [reply]
Thanx! 
At first glance it helped - more elements show correctly. But there was a problem with the menu items - not show cursor. what colors are used for the cursor? I can override them in terminal...

**Comment by Winsor** [comment]
ohh.... only XShell not show cursor in menu :( 
after set $TERM to xterm-256color putty&wezterm works nice. in XShell many many settings I tried to change, the ANSI color scheme is the same as in putty - nothing helps... menu cursor not show....

**Reply by BDisp** [reply]
What is the output by typing `echo $TERM` in the debian console?

**Reply by Winsor** [reply]
$ echo $TERM
xterm-256color

**Reply by BDisp** [reply]
Thanks. It's right, so is `XShell` issue that isn't allowing to use that settings as the `Putty` does. You may ask there. If you run directly on the debian console you can see the menu cursor, right? If is affirmative, then the `XShell` isn't allowing using the `xterm-256color` setting through ssh.
